src/qt/guiutil.cpp

```diff
@@ -243,6 +243,7 @@ QString HtmlEscape(const QString& str, bool fMultiLine)
     if(fMultiLine)
     {
         escaped = escaped.replace("\n", "<br>\n");
+        escaped = escaped.replace(" ", "&nbsp;");
```

Comment:
Can we achieve the same, non-collapsing spaces, using css' whitespace property? http://www.w3schools.com/cssref/pr_text_white-space.asp  . Or is this not supported by Qt?